ser/train.py

```diff
@@ -37,14 +42,20 @@ def train(name, learning_rate, epochs, batch_size):
 
     print("Data loaded properly.")
 
+    val_loss_temp = 1e10
+    correct_temp = 0
+    epoch_temp = 0
+    loss_function = nn.CrossEntropyLoss()
+
     # train
     for epoch in range(epochs):
         for i, (images, labels) in enumerate(training_dataloader):
```

Comment:
I think the training and validation logic should be split into separate functions.

It will make the overall set up and training loop code very simple, something like this:

```
def train():
   ...setup
   for epoch in range(epochs):
      for i, (images, labels) in enumerate(training_dataloader):
         train()
         validate()
```
Obviously there are details missing there, but the procedure is super easy to understand that way.

If train and val grow in complexity over time (which they probably will), the overall training code will remain easy to understand.